src/Styleguide/Components/ArtistBio.tsx

```diff
@@ -1,15 +1,25 @@
+import { ArtistBio_bio } from "__generated__/ArtistBio_bio.graphql"
+import { track } from "Analytics"
+import * as Schema from "Analytics/Schema"
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import { Responsive } from "Utils/Responsive"
 import { ReadMore } from "./ReadMore"
 
-import { ArtistBio_bio } from "__generated__/ArtistBio_bio.graphql"
-
 interface Props {
   bio: ArtistBio_bio
 }
 
+@track({ context_module: "Artist bio" })
 export class ArtistBio extends React.Component<Props> {
+  @track({
+    action_type: Schema.ActionTypes.Click,
+    action_name: Schema.ActionNames.ReadMoreExpanded,
+  })
```

Comment:
TODO: This should really move into the ReadMore component, or remain here if we need to keep using the artist specific `label`